qemu/tests/cfg/slof_boot.cfg

```diff
@@ -1,13 +1,74 @@
 - slof_boot: install setup image_copy unattended_install.cdrom
-    type = boot
+    virt_test_type = qemu
+    type = slof_boot
     restart_vm = yes
     kill_vm_on_error = yes
     login_timeout = 240
-    only ppc64 ppc64le
+    start_vm = yes
+    only ppc64le ppc64
+    only Linux
     variants:
-        - with_sec_size_4096_512:
-            physical_block_size = 4096
-            logical_block_size = 512
-        - with_sec_size_512_512:
-            physical_block_size = 512
-            logical_block_size = 512
+        - from_virtio_blk:
+            only virtio_blk
+            boot_dev_type = virtio_blk
+            # Set the address of virtio-blk-pci.
+            drive_pci_addr_image1 = 7
+            child_addr_virtio_blk = '${drive_pci_addr_image1}'
+            parent_bus_virtio_blk = 'pci'
+            child_bus_virtio_blk = 'scsi'
+            variants:
+                - @default:
+                - behind_pci_bridge:
+                    pci_controllers +=" pci_bridge1"
+                    type_pci_bridge1 = pci-bridge
+                    pci_bus_image1 = pci_bridge1
+                    child_bus_virtio_blk = 'pci-bridge'
+                    sub_child_addr_virtio_blk = '${drive_pci_addr_image1}'
+                - with_dataplane:
```

Comment:
It is better not to add dataplane variant here, since using `blk_extra_params` or `bus_extra_params` to specify iothread will be deprecated soon.

Reply:
I think this should talk about feature owner who could decide to how to design this case.